lib/show.js

```diff
@@ -11,7 +11,7 @@ var utils = require("./utils.js");
   - `show.def(x : a): string`
 */
 function showDef(obj) {
-  return "" + obj;
+  return JSON.stringify(obj);
```

Comment:
I don't actually know what you're using this for, so I should probably keep my mouth shut.

But is it acceptable that 1) this will drop some stuff, like functions, from the output, and 

2) that this will break in chrome if obj has circular references?